src/models.py

```diff
@@ -36,19 +31,21 @@ def check_password(self, password_to_check: str) -> bool:
         """Use this method to check User's password."""
         return self._get_password_hash(password_to_check) == self.password_hash
 
-    def create_user(self, username: str, password: str, session: scoped_session) -> None:
+    @classmethod
+    def create_user(cls, username: str, password: str) -> None:
         """Use this method to create a new user."""
-        self.username = username
-        self.set_password(password)
```

Comment:
Seems like `set_password` method is not used anywhere after we removed this line. Therefore we need to remove `set_password` method.